src/Objects/User.php

```diff
@@ -14,6 +20,57 @@
 class User extends BaseObject
 {
     /**
+     * @var string $currentConversation
+     */
+    private $currentConversation;
+
+    /**
+     * @var SQLite3Cache $cache
+     */
+    private $cache;
+
+    /**
+     * @inheritdoc
+     */
+    public function __construct($data)
+    {
+        parent::__construct($data);
+
+        $this->cache = new FilesystemCache(__DIR__.'/../Storage/cache');
+    }
+
+    /**
+     * @return string
+     */
+    public function getCurrentConversation()
+    {
+        if (is_null($this->currentConversation)) {
+            $cacheId = $this->getId() . '_current_conversation';
+            if (($currentConversation = $this->cache->fetch($cacheId)) !== false) {
+                $this->currentConversation = $currentConversation;
+
+                return $currentConversation;
+            };
+
+            $this->setCurrentConversation(HelloConversation::class);
```

Comment:
1. If you don't want to use conversation system, than you can disable it (has to be implemented). Or, we can make it disabled by default for now. Laravel users will be able to enable/disable it in telegram.php, and standalone will just need to call a method.
2. I will respond you with a solution today.